examples/models/plot_aperiodic_params.py

```diff
@@ -0,0 +1,177 @@
+
+###################################################################################################
+
+# Compute the correlation between the offset and exponent
+spearmanr(ap_values[0, :], ap_values[1, :])
+
+###################################################################################################
+#
+# What we see above matches the common finding that that the offset and exponent are
+# often highly correlated! This is because if you imagine a change in exponent as
+# the spectrum 'rotating' around some frequency value, then (almost) any change in exponent
+# has a corresponding change in offset value! If you note in the above, we actually specified
+# a rotation point around which the exponent changes.
+#
+# This can also be seen in this animation showing this effect across different rotation points:
+#
+# ![spectralrotation](https://raw.githubusercontent.com/fooof-tools/Visualizers/main/gifs/specrot.gif "specrot")
+#
+# Notably this means that while the offset and exponent can change independently (there can be
+# offset changes over and above exponent changes), the baseline expectation is that these
+# two parameters are highly correlated and likely reflect the same change in the data!
+#
+
+###################################################################################################
+# Knee Model
+# ----------
+#
+# Next, let's explore the knee model, which parameterizes the aperiodic component with
+# an offset, knee, and exponent value.
+#
+
+###################################################################################################
+
+# Generate a power spectrum with a knee
+freqs2, powers2 = gen_power_spectrum([1, 50], [0, 15, 1], [8, 0.125, 0.75], freq_res=0.25)
+
+###################################################################################################
+
+# Initialize model object and fit power spectrum
+fm = FOOOF(min_peak_height=0.05, aperiodic_mode='knee')
+fm.fit(freqs2, powers2)
+
+###################################################################################################
+
+# Plot annotated knee model
+plot_annotated_model(fm, annotate_peaks=False, annotate_aperiodic=True, plt_log=True)
+
+###################################################################################################
+
+# Check the measured aperiodic parameters
+fm.aperiodic_params_
+
+###################################################################################################
+# Knee Frequency
+# ~~~~~~~~~~~~~~
+#
+# You might notice that the knee _parameter_ is not an obvious value. Notably, this parameter
+# value as extracted from the model is something of an abstract quantify based on the
+# formalization of the underlying fit function. A more intuitive measure that we may
+# be interested in is the 'knee frequency', which is an estimate of the frequency value
+# at which the knee occurs.
+#
+# The `:func:`~.compute_knee_frequency` function can be used to compute the knee frequency.
```

Comment:
I think the first backtick needs to be remove in `:func:. I think it's preventing the linking from happening

Reply:
Ooops, yeh - good catch! Fixed